### PR TITLE
Add Niels Bohr Institute and their IceCube tier 3

### PR DESCRIPTION
**Pull request by matyasselmeci**
This is based on the info I got via email (and some Google Maps lookup).

### CODE REVIEW

File: topology/Niels Bohr Institute/NBI/NBI_T3.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Production: true
+SupportCenter: Self Supported
+
+GroupDescription: IceCube Tier 3 located at the Niels Bohr Institute
+GroupID: 509
+
+Resources:
+  NBI_T3_ARC_CE:
```
Comment (brianhlin) [inline topology/Niels Bohr Institute/NBI/NBI_T3.yaml]: Let's use the factory convention for the resource name (`<VO>_<COUNTRY CODE>_...`). I don't believe this is a hard requirement but this is how we've been registering the hosted CE resources.

```suggestion
  IceCube_DK_NBI_T3
```

Comment (matyasselmeci) [reply topology/Niels Bohr Institute/NBI/NBI_T3.yaml]: We should have CE in there though.

Comment (matyasselmeci) [reply topology/Niels Bohr Institute/NBI/NBI_T3.yaml]: IceCube_DK_NBI_T3_CE?

Comment (brianhlin) [reply topology/Niels Bohr Institute/NBI/NBI_T3.yaml]: Good point, I'll update the suggestion

**Comment by brianhlin** [comment]
Is this tagged as `do-not-merge` for any particular reason or can we go ahead and merge?

**Comment by matyasselmeci** [comment]
I sent mail to the NBI folks asking for some more information. We could either merge this now and make changes when they get back to me, or wait for them to get back to me first.

**Pull request merged**